staffjoy/resources/location.py

```diff
@@ -14,3 +15,13 @@ def get_role(self, id):
 
     def create_role(self, **kwargs):
         return Role.create(parent=self, **kwargs)
+
+    def get_managers(self):
+        return Manager.get_all(parent=self)
```

Comment:
Please add **kwargs here in case we add filters (I could see us adding some in the future)